firewall/interception/windowskext/service.go

```diff
@@ -0,0 +1,87 @@
+//go:build windows
+// +build windows
+
+package windowskext
+
+import (
+	"fmt"
+	"syscall"
+
+	"golang.org/x/sys/windows"
+)
+
+func createService(manager windows.Handle, portmasterKextPath *uint16) (windows.Handle, error) {
+	u16filename, err := syscall.UTF16FromString(driverName)
+	if err != nil {
+		return 0, fmt.Errorf("Bad service: %s", err)

```

Comment:
not sure if `Bad service` is a good error description.
How about `failed to convert name to UTF16 string: %w`?